genomicFileHandler/read_info_extractor.py

```diff
@@ -21,103 +21,103 @@
 def position_of_aligned_read(read_i, target_position):
     '''
     Return the base call of the target position, or if it's a start of insertion/deletion.
-    This target position follows pysam convension, i.e., 0-based. 
+    This target position follows pysam convension, i.e., 0-based.
     In VCF files, deletions/insertions occur AFTER the position.
-    
+
     Return (Code, seq_i, base_at_target, indel_length, nearest insertion/deletion)
-    
+
     The first number in result is a code:
     1) Match to reference, which is either a reference read or a SNV/SNP
     2) Deletion after the target position
     3) Insertion after the target position
-    0) The target position does not match to reference, and may be discarded for "reference/alternate" read count purposes, but can be kept for "inconsistent read" metrics. 
+    0) The target position does not match to reference, and may be discarded for "reference/alternate" read count purposes, but can be kept for "inconsistent read" metrics.
     '''
-    
+
     flanking_deletion, flanking_insertion = nan, nan
-    
+
     for i, align_i in enumerate(read_i.aligned_pairs):
-        
+
         # If find a match:
         if align_i[1] == target_position:
             seq_i = align_i[0]
             break
-    
-    
+
+
     # If the target position is aligned:
     try:
-        if isinstance(seq_i, int): 
+        if isinstance(seq_i, int):
```

Comment:
Is is always either `int` or `NoneType`? If yes, then, can simply use `if seq_i is not None`